apps/app/src/storage/localstorage/AsyncStorage.ts

```diff
@@ -0,0 +1,20 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+export const storeData = async <T>(value: T, key: string): Promise<boolean> => {
+  try {
+    const jsonValue = JSON.stringify(value);
+    await AsyncStorage.setItem(key, jsonValue);
+    return true;
+  } catch (e) {
+    throw new Error(`Not able to store data for key ${key}`);
+  }
+};
+
+export const getData = async <T>(key: string): Promise<T> => {
+  try {
+    const jsonValue = await AsyncStorage.getItem(key);
```

Comment:
**praise:** Actually very nice that you are wrapping your own functions around this library. Now it will be much easier to replace it with another library later. You'd just have to change this single file, because everywhere else in your code you are calling `storeData` and `getData` from this file, instead of the library directly.